e2e/helpers/pages/admin/posts/custom-view-modal.ts

```diff
@@ -0,0 +1,48 @@
+import {Locator, Page} from '@playwright/test';
+
+export class CustomViewModal {
+    private readonly page: Page;
+    public readonly modal: Locator;
+    public readonly nameInput: Locator;
+    public readonly nameError: Locator;
+    public readonly saveButton: Locator;
+    public readonly deleteButton: Locator;
+    public readonly cancelButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.modal = page.getByRole('dialog');
+        this.nameInput = page.getByLabel('View name');
+        this.nameError = page.locator('[data-test-error="custom-view-name"]');
```

Comment:
I assume later this one will end up being `page.getByTestId("error-customer-view")` or something like that